release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -652,6 +666,19 @@ This script removes unauthenticated subjects from the following cluster role bin
 +
 You can attempt to resolve the issue with the steps in this link:https://kb.vmware.com/s/article/2002779[VMware KBase article]. For more information, see the Red Hat Knowledgebase solution link:https://access.redhat.com/solutions/5785341[[UPI vSphere\] Node scale-up doesn't work as expected]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1918383[*BZ#1918383*])
 
+* If you are using {sandboxed-containers-first}, you cannot use the hostPath volume in a {product-title} cluster to mount a file or directory from the host node’s file system into your pod.
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1904609[*BZ#1904609*])
+
+* If you are running Fedora on {sandboxed-containers-first}, you need a workaround to install some packages. Some packages, like `iputils`, require file access permission changes that OSC does not grant to containers by default. To run containers that require such special permissions, it is necessary to add an annotation to the YAML file describing the workload, which tells `virtiofsd` to accept such file permissions for that workload. The required annotations are:
```

Comment:
s/OSC/(I don't recall what the non-"first" attribute is, but it should go here I think :smiley:)

Reply:
@zanetworker - OSC here is intended to be OpenShift sandboxed containers?

Reply:
@sagidlow I don't remember writing this, as I never use OSC :)

Maybe it's part of the initial sections in the overview that got moved to the release notes, we kind of were treating the RN section as a draft.

Reply:
I wrote this part, but I don't recall writing OSC either (maybe I wrote "Kata"?)

Reply:
Was this maybe @jensfr ?